examples/step-45/doc/intro.dox

```diff
@@ -1,72 +1,142 @@
 <br>
 
-<i>This program was contributed by Markus B&uuml;rg.</i>
-<br>
-
+<i>This program was contributed by Daniel Arndt and Matthias Maier.</i>
 <a name="Intro"></a>
 <h1>Introduction</h1>
 
-In this example we consider how to use periodic boundary conditions in
-deal.II. Periodic boundary conditions are a typical approach when one wants to
-solve some equation on a representative piece of a larger domain that repeats
-in one or more direction; an example is the simulation of the electronic
-structure of photonic
+In this example we present how to use periodic boundary conditions in
+deal.II. Periodic boundary conditions are algebraic constraints that
+typically occur in computations on representative regions of a larger
+domain that repeat in one or more directions.
+
+An example is the simulation of the electronic structure of photonic
 crystals, because they have a lattice-like structure and, thus, it often
-suffices to do the actual computation on only one cell of the lattice. To be
-able to proceed this way one has to assume that the computation can be periodically
-extended to the other cells. This requires the solution to be periodic with respect
-to the cells. Hence the solution has to obtain the same nodal values on opposite
-parts of the boundary. In the figure below we show this
-concept in two space-dimensions. There, all dashed faces with the same color should
-have the same boundary values:
+suffices to do the actual computation on only one cell of the lattice. To
+be able to proceed this way one has to assume that the computation can be
+periodically extended to the other cells; this requires the solution to
```

Comment:
Maybe mention somewhere that the "cell" you're talking about here has nothing to do with a finite element mesh cell.